packages/core/src/types/module-builder.ts

```diff
@@ -368,6 +386,31 @@ export interface IgnitionModuleBuilder {
     options?: StaticCallOptions
   ): StaticCallFuture<ContractNameT, FunctionNameT>;
 
+  /**
+   * Encode a function call.
```

Comment:
```suggestion
   * ABI encode a function call, including both the function's name and the parameters it is being called with. This is useful when sending a raw transaction to invoke a smart contract or when invoking a smart contract proxied through an intermediary function.
```

I am equivocal here. `Encode a function call` is a bit laconic. But I am not convinced I have improved on it. @zoeyTM what do you think?